src/holochainClient.js

```diff
@@ -119,6 +114,13 @@ let holochainClient
 let isInitiatingHcConnection = false
 let wsTimeoutErrorCount = 0
 
+// default wsConnetion to true in test env and hpos hp-admin env
+export let wsConnection = holochainClient
+  ? true
+  : process.env.NODE_ENV === 'test'
+    ? true
+    : !process.env.REACT_APP_HOLOFUEL_APP
```

Comment:
we should default this value to true in all cases. Let the error checking mechanism set it to false. Otherwise we have to add a bunch of extra logic in the UI.